corehq/apps/hqwebapp/static/hqwebapp/js/inactivity.js

```diff
@@ -0,0 +1,115 @@
+/*
+ *  Handles inactvitiy timeout UI: a modal containing an iframe with the login screen,
+ *  allowing users to re-login without leaving the page and losing their work.
+ */
+hqDefine('hqwebapp/js/inactivity', [
+    'jquery',
+    'underscore',
+    'hqwebapp/js/initial_page_data',
+], function (
+    $,
+    _,
+    initialPageData
+) {
+    $(function () {
+        var timeout = initialPageData.get('secure_timeout') * 60 * 1000,    // convert from minutes to milliseconds
+            $modal = $("#inactivityModal"),     // won't be present on app preview or pages without a domain
```

Comment:
Can these comments be verified with tests?

Reply:
The way I'd test it would be to call the view and then check the generated HTML with a regex for this id, which seems like a slow and fragile for a fairly minor test case. Any thoughts on a better way to test this?